guides/common/modules/proc_using-ansible-provider-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Using_Ansible_Provider_for_Package_and_Errata_Actions_{context}"]
+= Using Ansible provider for package and errata actions
+
+By default, {Project} is configured to use the Script provider templates for remote execution jobs.
+If you prefer using the Ansible provider templates for your remote jobs, you can configure {Project} to use the Ansible provider templates for remote execution features that take a search query input.
```

Comment:
Okaaay how about this?

```suggestion
If you prefer using Ansible job templates for your remote jobs, you can configure {Project} to use them by default for remote execution features associated with them.
```

I originally had "features that support them," but I think it's more accurate to say that a template "supports" a feature, rather than the other way around, since you specify the `feature: xxxx` in the template's YAML header.